example/full/instance.properties

```diff
@@ -378,6 +378,48 @@ sleeper.bulk.import.emr.ebs.volume.type=gp2
 # This can be a number from 1 to 25.
 sleeper.bulk.import.emr.ebs.volumes.per.instance=4
 
+# The class to use to perform the bulk import.
+sleeper.bulk.import.emr.serverless.class.name=sleeper.bulkimport.job.runner.dataframelocalsort.BulkImportDataframeLocalSortDriver
+
+# The architecture for EMR Serverless to use. X86_64 or ARM (Coming soon)
+sleeper.bulk.import.emr.serverless.architecture=X86_64
+
+# The switch to enable EMR Serverless over persistent EMR.
+sleeper.bulk.import.emr.serverless.enabled=true
+
+# The version of EMR Serverless to use.
+sleeper.bulk.import.emr.serverless.release=emr-6.10.0
+
+# The name of the repository for the EMR serverless container. The Docker image from the bulk-import
+# module should have been uploaded to an ECR repository of this name in this account.
+sleeper.bulk.import.emr.serverless.repo=<insert-unique-sleeper-id>/bulk-import-runner-emrs
```

Comment:
I think it would be clearer if `bulk-import-runner-emrs` was `bulk-import-runner-emr-serverless`

Reply:
It has been renamed